source/operations/install-deploy-manage/migrate-fs-gateway.rst

```diff
@@ -0,0 +1,203 @@
+.. _minio-gateway-migration:
+
+=======================================
+Migrate from Gateway or Filesystem Mode
+=======================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Background
+----------
+
+The MinIO Gateway and the related filesystem mode entered a feature freeze in July 2020.
+In February 2022, MinIO announced the `deprecation of the MinIO Gateway <https://blog.min.io/deprecation-of-the-minio-gateway/?ref=docs>`__.
+Along with the deprecation announcement, MinIO also announced that the feature would be removed in six months time.
+
+As of :minio-release:`RELEASE.2022-10-29T06-21-33Z`, the MinIO Gateway and the related filesystem mode code have been removed.
+Deployments still using the `standalone` or `filesystem` MinIO modes that upgrade to :minio-release:`RELEASE.2022-10-29T06-21-33Z` or later receive an error when attempting to start MinIO.
+
+Overview
+--------
+
+To upgrade to a :minio-release:`RELEASE.2022-10-29T06-21-33Z` or later, those who were using the `standalone` or `filesystem` deployment modes must create a new :ref:`Single-Node Single-Drive <minio-snsd>` deployment and migrate settings and content to the new deployment.
+
+This document outlines the steps required to successfully launch and migrate to a new deployment.
+
+.. important:: 
+
+   Standalone/file system mode continues to work on any release up to and including `RELEASE.2022-10-24T18-35-07Z <https://github.com/minio/minio/releases/tag/RELEASE.2022-10-24T18-35-07Z>`__.
+   To continue using a standalone deployment, install that MinIO release or any `earlier release <https://github.com/minio/minio/releases>`__.
+
+   Do not upgrade to any release later than ``RELEASE.2022-10-24T18-35-07Z``.
+
+Procedure
+---------
+
+#. Retrieve the existing deployment's **environment variables**
+
+   Use the :mc-cmd:`mc admin config` export command to retrieve the environment variables defined on the existing standalone MinIO deployment.
```

Comment:
```suggestion
   Use the :mc-cmd:`mc admin config export <mc admin config>`  command to retrieve the environment variables defined on the existing standalone MinIO deployment.
```

Reply:
I see the command is missing - I would say just add a stub for now on the mc admin config page that just says

```
mc admin config export
- exports any configuration settings created using `mc admin config set`
```

Since as @eco-minio noted previously anything envvar set does not necessarily end up in this data (iirc)

Reply:
👍